dts/bindings/pwm/nordic,nrf-sw-pwm.yaml

```diff
@@ -5,19 +5,32 @@ compatible: "nordic,nrf-sw-pwm"
 include: [pwm-controller.yaml, base.yaml]
 
 properties:
+    rtc-instance:
+      type: int
```

Comment:
Why not phandle and pointing to the `nordic,nrf-rtc` directly? I think it should probably be done that way, just looking at this quickly.